site/public/js/admin-gradeable-updates.js

```diff
@@ -286,13 +325,20 @@ function setRandomGraders(gradeable_id,p_values,successCallback,errorCallback) {
       return false;} 
     }
     var gradeable_id=$('#g_id').val();
+    var restrict_to_registration="unchecked";
     $('#peer_loader').removeClass("hide");
+    if($('#restrict_to_registration').is(':checked')){
+        restrict_to_registration="checked";
+    }
+        
     $.ajax({
         type: "POST", 
         url: buildCourseUrl(['gradeable', gradeable_id, 'RandomizePeers']),
-        data: {csrf_token:p_values['csrf_token'],number_to_grade:number_to_grade},
+        data: {csrf_token:p_values['csrf_token'],number_to_grade:number_to_grade,restrict_to_registration:restrict_to_registration},
```

Comment:
put the dictionary values on their own line so its easier to read
```json
data:{
  "csrf_token" : p_values[...],
  "nuber_to_grade" : ...,
  ...
}
```

Reply:
corrected!